django/contrib/postgres/fields/array.py

```diff
@@ -179,6 +181,19 @@ def as_sql(self, qn, connection):
 
 
 @ArrayField.register_lookup
+class ArrayIn(In):
+    def batch_process_rhs(self, compiler, connection, rhs=None):
+        sqls, sqls_params = [], []
+        for p in rhs:
+            value = QueryWrapper(
```

Comment:
I don't think it has a lot of utility, and could probably be replaced or reimplemented as a proper expression if there was ever the need. It seems to be just a light weight type for passing sql, param tuples around. Let's leave it be for now.